nss-cohort-36/keahua-arboretum-mythical-techies
# Sh interface functions

# Description

adding a biome to arboretum dictionary

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

git fetch --all
git checkout sh-interface-functions
see if you can run index.py and add river biome (can be done multiple times)


# Checklist:
